scripts/_common.sh

```diff
@@ -46,3 +46,9 @@ if ! output=$(
 
   exit 1
 fi
+
+if [[ ! -f submodules/stack-graphs/stack-graphs/Cargo.toml ]]; then
```

Comment:
this should already be taken care of by `infra setup git` command.

Reply:
This doesn't work, because to build `infra` we need to have all the dependencies available, and stack-graph being in a submodule means it's not.

Reply:
I see. Thanks for explaining!